### PR TITLE
server: abandon support for raftcmd from network.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

**Comment by siddontang** [comment]
Do we need to remove raftcmd from kvproto too?

**Comment by disksing** [comment]
I think we can update kvproto at the same time with grpc.

**Comment by siddontang** [comment]
PTAL @BusyJay

**Comment by siddontang** [comment]
PTAL @BusyJay

File: src/server/server.rs [hunk 1]
```diff
@@ -255,10 +254,7 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
                 try!(self.ch.raft_router.send_raft_msg(msg.take_raft()));
                 Ok(())
             }
-            MessageType::Cmd => {
-                RECV_MSG_COUNTER.with_label_values(&["cmd"]).inc();
-                self.on_raft_command(msg.take_cmd_req(), token, msg_id)
-            }
+            MessageType::Cmd => panic!("invalid message type raftCmd"),
```
Comment (BusyJay) [inline src/server/server.rs]: I think close the connection or return error is more appropriate. So the tikv server won't crash due to invalid message sent by client.

**Comment by disksing** [comment]
PTAL @BusyJay

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**